backend/src/core/services/file-attachment.service.ts

```diff
@@ -35,19 +37,45 @@ const parseFiles = async (
 }
 
 const sanitizeFiles = async (
-  files: { data: Buffer; name: string }[]
+  files: { data: Buffer; name: string }[],
```

Comment:
at some point, probably good to align the various variable names (`files` and `attachments`) and type (think there is a `UploadedFile` type we could use that's more informative)

Reply:
We can possibly use `ReadableStream` type for these I think

Reply:
TypeScript TypeScript TypeScript